README.md

```diff
@@ -1,10 +1,10 @@
 <picture>
-  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/eclipse-thingweb/website/master/misc/thingweb_logo_for_dark_bg.svg">
-  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/eclipse-thingweb/website/master/misc/thingweb_logo.svg">
-  <img title="ThingWeb" alt="Thingweb logo" src="https://raw.githubusercontent.com/eclipse-thingweb/website/master/misc/thingweb_logo.svg" width="300">
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/eclipse-thingweb/thingweb/master/brand/logos/node-wot_for_dark_bg.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/eclipse-thingweb/thingweb/master/brand/logos/node-wot.svg">
+  <img title="ThingWeb node-wot" alt="Thingweb node-wot logo" src="https://raw.githubusercontent.com/eclipse-thingweb/thingweb/master/brand/logos/node-wot.svg" width="300">
 </picture>
 
-## node-wot
+##
```

Comment:
Hmm, having an empty heading here seems not that ideal to me here. If we want to remove the duplication of "node-wot", I think we could also consider wrapping the `picture` element in an `h2` or `h1` instead.

Reply:
Leaving it empty doesn't seem right to me either. What about removing the line

`## node-wot`

entirely?